File: Java/Program/Product.java
public class Product{
    private int idProduct, price;
    private String name, brand;

    public Product(){
        this.idProduct = -1;
        this.price = 0;
        this.name = "";
        this.brand = "";
    }

    public Product(int idProduct, String name, String brand, int price){
        this.idProduct = idProduct;
        this.price = price;
        this.name = name;
        this.brand = brand;
    }

    public void setIdProduct(int idProduct) {
        this.idProduct = idProduct;
    }
    
    public void setPrice(int price) {
        this.price = price;
    }
    
    public void setName(String name) {
        this.name = name;
    }
    
    public void setBrand(String brand) {
        this.brand = brand;
    }
    
    public int getIdProduct() {
        return idProduct;
    }
    
    public int getPrice() {
        return price;
    }
    
    public String getName() {
        return name;
    }
    
    public String getBrand() {
        return brand;
    }   
}

File: Python/program/Product.py
class Product:
    __idProduct = -1
    __name = ""
    __brand = ""
    __price = 0

    def __init__(self, idProduct = -1, name = "", brand = "", price = 0):
        self.__idProduct = idProduct
        self.__name = name
        self.__brand = brand
        self.__price = price

    def getIdProduct(self):
        return self.__idProduct
    
    def getName(self):
        return self.__name
    
    def getBrand(self):
        return self.__brand
    
    def getPrice(self):
        return self.__price

    def setIdProduct(self, idProduct):
        self.__idProduct = idProduct
    
    def setName(self, name):
        self.__name = name
    
    def setBrand(self, brand):
        self.__brand = brand
    
    def setPrice(self, price):
        self.__price = price